### DISCUSSION TITLE
user can only request a Magic Link once every 60 seconds

### DISCUSSION THREAD

**Discussion by RyanTippsTX** [supabase/supabase#9294] [Questions]
[Docs](https://supabase.com/docs/reference/javascript/auth-signin) say this, but they don't explain how to change it.

I would like ~5 or 10 seconds. How can I do this?

**Comment by GaryAustin1** [comment]
That is a configuration parameter for gotrue and there is no access from the dashboard on hosted to change it for Supabase's mailer.

gotrue https://github.com/supabase/gotrue/tree/master/conf/configuration.go
```
	if config.SMTP.MaxFrequency == 0 {
		config.SMTP.MaxFrequency = 1 * time.Minute
	}
```
If you use your own SMTP service though, you can change it:
![image](https://user-images.githubusercontent.com/54564956/193606600-7d9abd88-b472-48e0-b7a1-ab4b27b6f5ff.png)
